benchmarks/operator_benchmark/pt/qinterpolate_test.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import operator_benchmark as op_bench
+import torch
+
+'''Microbenchmarks for the quantized interpolate op.
+
+Note: We are not benchmarking `upsample` as it is being depricated, and calls
+the `interpolate` anyway.
+'''
+
+# Configs for PT add operator
+qinterpolate_long_configs = op_bench.config_list(
+    attr_names=['M', 'N', 'K'],
+    attrs=[
+        [512, 512, 512],
+    ],
+    cross_product_configs={
+        'dtype': [torch.quint8, torch.qint8, torch.qint32],
+        'mode': ['nearest', 'bilinear'],
+        'scale': [0.25, 0.5, 1.0, 1.5, 2.0],
+    },
+    tags=['long']
+)
+
+
+qinterpolate_short_configs = op_bench.config_list(
+    attr_names=['M', 'N', 'K', 'dtype', 'mode', 'scale'],
+    attrs=[
+        [32, 32, 32, torch.quint8, 'nearest', 0.5],  # Downsample
+        [32, 32, 32, torch.quint8, 'bilinear', 0.5],  # Downsample
+        [32, 32, 32, torch.quint8, 'nearest', 2.0],  # Upsample
+        [32, 32, 32, torch.quint8, 'bilinear', 2.0],  # Upsample
+    ],
+    tags=['short'],
+)
+
+
+class QInterpolateBenchmark(op_bench.TorchBenchmarkBase):
+    def init(self, M, N, K, dtype, mode, scale):
+        f_input = (torch.rand(1, M, N, K) - 0.5) * 256
+        scale = 0.1
+        zero_point = 64
```

Comment:
Just want to check if zero_point is intentionally set to 64? As I saw it was set to 0 for other tests.

Reply:
Yes, my logic was in case there is a fast pass in the interpolation logic, where everything is 0, the behavior might be different. However, if the dtype is `quint8`, all the negative numbers might become 0, which will take a fast pass. To avoid that, I am setting the zero_point to some non-zero value.

I cannot set it to 128, as that would have the same (potential) problem with `qint8`, hence 64... I guess I should set it to 42 :)